static/app/views/app/index.tsx

```diff
@@ -42,10 +42,8 @@ function App({children}: Props) {
     [
       {
         match: ['command+shift+p', 'command+k', 'ctrl+shift+p', 'ctrl+k'],
-        callback: e => {
-          openCommandPalette();
-          e.preventDefault();
-        },
+        includeInputs: true,
+        callback: () => openCommandPalette(),
```

Comment:
extra closure?

Reply:
Callback passes the event, but openCommandPalette takes a different optional arg